apps/betterangels-backend/accounts/utils.py

```diff
@@ -16,15 +17,27 @@ def remove_organization_permission_group(organization: Organization) -> None:
 
 
 def add_default_org_permissions_to_user(user: User, organization: Organization) -> None:
-    default_permission_group, _ = PermissionGroupTemplate.objects.get_or_create(
+    caseworker_permission_group, _ = PermissionGroupTemplate.objects.get_or_create(
         # TODO: This is a hack for MVP. Not all orgs will default to caseworkers
         # we will want to have a default template selected for orgs on the org model.
         name=GroupTemplateNames.CASEWORKER
     )
-    org_permission_group, _ = PermissionGroup.objects.get_or_create(
-        organization=organization, template=default_permission_group
+    org_caseworker_group, _ = PermissionGroup.objects.get_or_create(
+        organization=organization, template=caseworker_permission_group
     )
-    user.groups.add(org_permission_group.group)
+    user.groups.add(org_caseworker_group.group)
+
+
+def create_default_org_permission_groups(organization: Organization) -> None:
+    default_templates = [
```

Comment:
should probably add a note to `GroupTemplateNames` stating that this util should be updated whenever we add a template for caseworker orgs